src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -185,17 +185,206 @@
           "application/json"
         ],
         "parameters": [],
-        "produces": ["application/vnd.schemaregistry.v1+json"],
+        "produces": [
+          "application/vnd.schemaregistry.v1+json",
+          "application/vnd.schemaregistry+json",
+          "application/json"
+        ],
         "responses": {
           "200": {
             "description": "OK",
             "schema": {
-              "type": "object",
-              "properties": {
-                "mode": {
-                  "type": "string"
-                }
-              }
+              "$ref": "#/definitions/mode"
+            }
+          },
+          "500": {
+            "description": "Internal Server error",
+            "schema": {
+              "$ref": "#/definitions/error_body"
+            }
+          }
+        }
+      },
+      "put": {
+        "summary": "Set the global mode.",
+        "operationId": "put_mode",
+        "consumes": [
+          "application/vnd.schemaregistry.v1+json",
+          "application/vnd.schemaregistry+json",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "mode",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/mode"
+            }
+          }
+        ],
+        "produces": [
+          "application/vnd.schemaregistry.v1+json",
+          "application/vnd.schemaregistry+json",
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/mode"
+            }
+          },
+          "422": {
+            "description": "Unprocessable Entity",
+            "schema": {
+              "$ref": "#/definitions/error_body"
+            }
+          },
+          "500": {
+            "description": "Internal Server error",
+            "schema": {
+              "$ref": "#/definitions/error_body"
+            }
+          }
+        }
+      }
+    },
+    "/mode/{subject}": {
+      "get": {
+        "summary": "Get the mode for a subject.",
+        "operationId": "get_mode_subject",
+        "consumes": [
+          "application/vnd.schemaregistry.v1+json",
+          "application/vnd.schemaregistry+json",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subject",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "defaultToGlobal",
```

Comment:
From the description for the query param
> query the mode for a particular subject, regardless of whether it has a specific override
and the endpoint (without the query param)
> If there is no specific mode set for the subject an error shall be returned.
Does `defaultToGlobal` mean, if there is no specific mode set for the subject, return the global mode instead? I wasn't sure what "specific override" was referring to

Reply:
> From the description for the query param
> > query the mode for a particular subject, regardless of whether it has a specific override
> 
> and the endpoint (without the query param)
> > If there is no specific mode set for the subject an error shall be returned.
> 
> Does `defaultToGlobal` mean, if there is no specific mode set for the subject, return the global mode instead? I wasn't sure what "specific override" was referring to

Yeah, so the global mode is in force for all subjects, unless a subject has a mode set, in which case that will override whatever is global, for the subject.

To query whether a subject has a mode set, either don't specify `defaultToGlobal=true`, or specify `defaultToGlobal=false`

But if you're just querying what mode is in force, specify `defaultToGlobal=true`

Reply:
@kbatuigas I added some short descriptions for the parameters.